### ISSUE TITLE
Failed at the helios-weather-app@1.0.0 dev script

### ISSUE THREAD

**Issue by bryantee** [rockchalkwushock/Helios-Weather-App#2]
Getting error during `npm-start`:
```
npm ERR! Tell the author that this fails on your system:
npm ERR!     webpack-dev-server --inline --hot --content-base build --history-api-fallback
```

I've attached the npm-debug.log if it's of any help.

[npm-debug.txt](https://github.com/rockchalkwushock/Helios-Weather-App/files/593086/npm-debug.txt)

**Comment by rockchalkwushock** [comment]
I will look in to this more tomorrow but this is what I noticed on the fly:

1. Your system is running on `npm --v 4.0.0` while mine runs on `npm --v 3.10.9`.
2. Your system is running on `node -v 6.9.1` while mine runs on `node -v 7.1.0`.
3. In your `PATH` on line 9 it goes about mentioning `heroku` which is interesting because my project isn't hosted on `heroku`.

**NOTE**: I did just totally remaster my system from the ground up so no projects were present on my drive and everything is at it's base values and/or initial installs. I cloned the repo to my machine, followed the instructions in the `README.md` and it worked with no issues. I am wondering if it is an issue with versions of `node` & `npm`. I will create a throw aways OS in VirtualBox and install the specific versions your system is running on and test tomorrow. If it is a version specific problem I will make note of it in the `README.md` for future reference.

Thanks for reporting the issue!
